src/views/modals/ModalTransactionConfirmation/ModalTransactionConfirmationTs.ts

```diff
@@ -742,6 +760,29 @@ export class ModalTransactionConfirmationTs extends Vue {
     }
 
     private async ledgerAccMultisigDelHarvestKeyOnStop(values) {
+        const accountAddress = this.command.currentSignerHarvestingModel.accountAddress;
+
+        // store new vrf Key info in local
+        if (this.command.vrfPrivateKeyTemp) {
+            this.saveVrfKeyInfo(
+                accountAddress,
+                Crypto.encrypt(this.command.newVrfKeyAccount.privateKey, this.command.password),
+                this.command.newVrfKeyAccount.publicKey,
+            );
+        }
+
+        // store new remote Key info in local
+        if (this.command.remotePrivateKeyTemp) {
+            this.saveRemoteKeyInfo(
+                accountAddress,
+                Crypto.encrypt(this.command.newRemoteAccount.privateKey, this.command.password),
+                this.command.newRemoteAccount.publicKey,
+            );
+        }
+
+        // // pre-store selected harvesting node in local
+        this.saveHarvestingNode(accountAddress, this.command.formItems.nodeModel);
+
```

Comment:
This block looks like the block (L579-601), could we extract to a method and reuse that method?

Reply:
yes, that is possible. Was thinking about where should we put this method.
HarvesrtingServices?

Reply:
I think a private method within the same file would be enough for now.